tests/test_frameworks.py

```diff
@@ -286,3 +295,48 @@ def forward(self, X):
     cl = CleanLearning(net)
     cl.fit(dataset, data["y"], clf_kwargs={"epochs": 2})
     pred_probs = cl.predict(dataset)
+
+
+# test fasttext
+@pytest.mark.skipif(
+    "not run_fasttext_test()", reason="fasttext is not easily pip install-able on windows"
+)
+def test_fasttext():
+    from cleanlab.models.fasttext import FastTextClassifier, data_loader
+
+    dir = "tests/fasttext_data"
+    if not os.path.isdir(dir):
+        os.makedirs(dir)
+
+    if not os.path.isfile("tests/fasttext_data/tweets_train.txt"):
```

Comment:
how long does it take to run this dataset? Any way to use a smaller version of the dataset so that the test will run quicker?

Reply:
Including the download, this unittest takes about 5 or 6 seconds. I can make the dataset smaller but I'm not sure if it will cut down the time by much?

Reply:
5-6 sec is fine! I wasn't sure.
Could you add a print statement that clarifies how developer can skip this unit test (via env variable) in the case where the download fails (eg. because no internet connection)